packages/postgres-database/src/simcore_postgres_database/utils_folders.py

```diff
@@ -329,39 +339,35 @@ async def _get_resolved_access_rights(
     gid: _GroupID,
     *,
     permissions: _FolderPermissions | None,
-    enforece_all_permissions: bool,
+    enforce_all_permissions: bool = False,
```

Comment:
This was only used in tests, and it, made no sense to test a more restrictive use case. I've dropped this, since it was a leftover from the initial design.